crates/bevy_core_pipeline/src/experimental/mip_generation/downsample_depth.wgsl

```diff
@@ -1,8 +1,16 @@
 #ifdef MESHLET_VISIBILITY_BUFFER_RASTER_PASS_OUTPUT
 @group(0) @binding(0) var<storage, read> mip_0: array<u64>; // Per pixel
```

Comment:
I believe all the meshlet-specific stuff is going to disappear here once wgpu 24 is merged and I can switch back to image atomics.